content-src/components/DiscoveryStreamComponents/DSContextFooter/DSContextFooter.jsx

```diff
@@ -34,7 +40,7 @@ export class DSContextFooter extends React.PureComponent {
               timeout={ANIMATION_DURATION}
               classNames="story-animate"
             >
-              {engagement && !context_type ? (
+              {display_engagement_labels && engagement && !context_type ? (
```

Comment:
Adding display_engagement_labels here will always return false when display_engagement_labels is false irrespective of we want to show context_type (e.g. 'Bookmarked') .

Seeing below in DSCard with the patch when engagement label is hidden.

<img width="602" alt="status-message" src="https://user-images.githubusercontent.com/597734/65924884-827e4b00-e3a3-11e9-9904-964d9fb78890.png">

I propose moving check above to Line 37 e.g.

`!context && (context_type || (display_engagement_labels  && engagement))`

so that this code is entered when we have either context_type or engagement to show.